client/components/header-nav/README.md

```diff
@@ -0,0 +1,4 @@
+Header Nav
```

Comment:
Ah ok. That's just a way to handle responsiveness, I wouldn't make it a characteristic trait. I think its defining properties are being a navigation item, being attached to the masterbar. :)